src/main/java/cz/cvut/kbss/study/security/SecurityConstants.java

```diff
@@ -27,7 +27,24 @@ private SecurityConstants() {
      */
     public static final int SESSION_TIMEOUT = 12 * 60 * 60;
 
+    public static final String ROLE_GROUP_OPERATOR_ADMIN = "ROLE_GROUP_OPERATOR_ADMIN";
+    public static final String ROLE_GROUP_OPERATOR = "ROLE_GROUP_OPERATOR";
+    public static final String ROLE_GROUP_SUPPLIER_ADMIN = "ROLE_GROUP_SUPPLIER_ADMIN";
+    public static final String ROLE_GROUP_SUPPLIER = "ROLE_GROUP_SUPPLIER";
+    public static final String ROLE_GROUP_EXTERNAL_USER = "ROLE_GROUP_EXTERNAL_USER";
     public static final String ROLE_USER = "ROLE_USER";
-
     public static final String ROLE_ADMIN = "ROLE_ADMIN";
+    public static final String ROLE_COMPLETE_RECORDS = "rm_complete_records";
```

Comment:
Why do we need those constants? Why Role enum is not enough for this case? (same way as we discussed in frontend)